docs/getting_started.md

```diff
@@ -68,14 +68,15 @@ Fog::OpenStack.services
 These services can be explored in further depth in the following documents:
 
 * [Compute](compute.md)
+* [Introspection](introspection.md)
+* [Metering](metering.md)
+* [Network](network.md)
+* [NFV](nfv.md)
+* [Orchestration](orchestration.md)
+* [Planning](planning.md)
+* [Share (Manila)](share.md)
```

Comment:
Documentation is effectively behind, thank you for doing this.
Meanwhile could you please make those 'common' changes into a separate Pull Request?
So we can keep track of changes.